core/src/AbilityHelpers.ts

```diff
@@ -170,7 +170,14 @@ class AbilityHelpers implements Types.AbilityHelpersCore, Types.AbilityHelpersIn
 }
 
 export function createAbilityHelpers(win: Window, props?: Types.AbilityHelpersCoreProps): Types.AbilityHelpersCore {
```

Comment:
Also we need to delete the instance when disposeAbilityHelpers is called.

Reply:
Added this to `dispose` member of `AbilityHelpers`